frontend/src/components/smart/SkillBalanceDisplay.vue

```diff
@@ -109,6 +109,31 @@
         </div>
       </div>
     </div>
+    <b-modal
+      class="centered-modal"
+      ref="minting-sfuel-modal"
+      hide-footer
+      hide-header
+      hide-header-close
+      no-close-on-backdrop
+      no-close-on-esc
+      centered
+    >
+      <div class="text-center">
+        <div class="mb-2">
+          <div class="spinner-grow text-center" style="width: 3rem; height: 3rem;" role="status">
```

Comment:
Please don't use inline styles whenever possible, best do a class and use a css class here. Also, did you check if it's responsive? Does it look ok on mobile?